model/trace/faas.yaml

```diff
@@ -33,18 +33,6 @@ groups:
       - ref: faas.document.time
       - ref: faas.document.name
 
-  - id: faas_span.http
-    type: span
-    brief: >
-      Semantic Convention for FaaS triggered as a response to some data
-      source operation such as a database or filesystem read/write.
-
-  - id: faas_span.pubsub
-    type: span
-    brief: >
-      Semantic Convention for FaaS set to be executed when messages are
-      sent to a messaging system.
```

Comment:
All of the rest of the changes are formating.  This removes some spans.   Is there a reason for this?

Reply:
We should notify the FAAS group, likely.

I agree on removing and adding it back in when they're documented.

Reply:
Thanks for the heads up.  I'm good with removing them and revisiting later.